test/xml-generator/ParameterFrameworkConfiguration.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ParameterFrameworkConfiguration SystemClassName="Test" ServerPort="5066" TuningAllowed="false">
+    <SubsystemPlugins>
```

Comment:
I used UpperCamelCase for the files that are used directly by the parameter framework instance started by the test suite because that's how we have done it historically. I've used a "free format" for the files that are imported by the xml generator.